mpas_analysis/shared/mpas_xarray/mpas_xarray.py

```diff
@@ -141,59 +141,114 @@ def ensure_list(alist):  # {{{
     return alist  # }}}
 
 
-def get_datetimes(ds, timestr, yearoffset):  # {{{
+def parse_time(ds, inTimeVarName, inReferenceDate, outTimeVarName,
+               outReferenceDate):  # {{{
     """
-    Computes a list of datetimes from the time variable in the dataset ds with
-    variable name (or a list of 2 names) given by timestr, typically one of
-    'daysSinceStartOfSim', 'xtime', or ['xtime_start', 'xtime_end'].
+    Converts MPAS times to `netcdftime.DatetimeNoLeap` objects consistent with
+    MPAS's 365-day calendar.
```

Comment:
@pwolfram, okay, great.  Thanks for the comment.  That should mean that the calling code should be able to figure out which calendar to use without the need for any config variables.  It's also good to know that there *are* multiple calendar options.  I should have looked into this myself.